### PR TITLE
Some suggested minor edits

### PR DESCRIPTION
**Pull request by fcooper8472**
Mainly for language, readability, markdown style etc.

### CODE REVIEW

**Comment by fcooper8472** [comment]
I realise it would be confusing to open an issue that isn't about code review, so a couple of things here:

Each heading can be directly linked to (e.g. https://github.com/OxfordCodeReviewNet/forum#-what-are-code-reviews) so you don't need all the explicit html links.

The C++ core guidelines referenced here (https://github.com/OxfordCodeReviewNet/forum/blob/master/guidelines_for_reviewers.md#code-style) would be better in a static analysis section.  For code formatting you want [clang format](https://clang.llvm.org/docs/ClangFormat.html) which has built-in styles (that can be customised for your project) for LLVM, Google, Chromium, Mozilla, WebKit, Microsoft.

**Comment by tlestang** [comment]
Thanks for the thorough review! Really helpful.
Re the links, I still prefer to keep them internal to the Markdown doc, i.e. independent of the final web address of the document. Anything against that ?

File: guidelines_for_reviewers.md [hunk 1]
```diff
@@ -20,20 +20,19 @@ If you have no idea what to look for, here is a list of the major points to chec
...
-5. [Obscure lines](#obscure)
-4. [Unintended behaviour](#unintended)
-5. [Comments](#comments)
-6. [Performance low hanging fruits](#performance)
-7. [Potential use of built-in functions](#builtin)
-8. [Potential use of third-party libraries](#third-party)
-
+1. [Duplicated code](#duplicated)
+1. [Long functions](#long)
+1. [Complex if statements](#complexity)
```
Comment (tlestang) [inline guidelines_for_reviewers.md:25]: Could you just fix the indices?

File: remote.md [hunk 2]
```diff
@@ -98,14 +108,17 @@ Floobits provides useful features for code reviews and pair programming in gener
...
-3. Files on work-spaces are read-only by default. To grant write permission to another user, go to your Floobits settings page and update your work-space's settings: *Settings* -> *Edit Permissions*.
-4. You can join any public workspace (or private if you were added to it) from your text editor. To join a workspace from the web-based editor, the workspace owner must first add you to it. Then in your Floobits profile, click on *Edit code*.
+
+1. Create an account on [https://floobits.com/](https://floobits.com/)
+1. To share your code, create a workspace and synchronise your local files with it. From the web-editor, *File* -> *Add files*. This can also be done from your local editor.
+1. Files on work-spaces are read-only by default. To grant write permission to another user, go to your Floobits settings page and update your work-space's settings: *Settings* -> *Edit Permissions*.
+1. You can join any public workspace (or private if you were added to it) from your text editor. To join a workspace from the web-based editor, the workspace owner must first add you to it. Then in your Floobits profile, click on *Edit code*.
```
Comment (tlestang) [inline remote.md:115]: Same above

**Review by tlestang — CHANGES_REQUESTED** [review]
Looks great. Some lists only contain 1s as indices though. If you could just change that and I'll merge the PR

**Comment by fcooper8472** [comment]
> Thanks for the thorough review! Really helpful.
> Re the links, I still prefer to keep them internal to the Markdown doc, i.e. independent of the final web address of the document. Anything against that ?

No, that's fine!

File: guidelines_for_reviewers.md (hunk 1 above)
Comment (fcooper8472) [reply guidelines_for_reviewers.md:25]: Not sure what you mean?

**Comment by fcooper8472** [comment]
> Looks great. Some lists only contain 1s as indices though. If you could just change that and I'll merge the PR

Ah, I see.  I think that's completely standard Markdown notation.  You list every item with a 1. and markdown figures out the appropriate number.  So if you ever want to insert an item in the middle of the list you don't have to re-number.

**Comment by tlestang** [comment]
Ah ok! Makes sense! (I write in org-mode, which usually renumbers on the fly)

**Review by tlestang — APPROVED** [review]

**Pull request merged**